vcsim/README.md

```diff
@@ -6,6 +6,19 @@ language, it can be used by any language that can talk to the vSphere API.
 
 ## Installation
 
+### Docker
+
+The official `vcsim` [Docker images](https://hub.docker.com/r/vmware/vcsim) are built from this [Dockerfile](../Dockerfile.vcsim).
+
+### Binaries
+You can find prebuilt `vcsim` binaries on the [releases page](https://github.com/vmware/govmomi/releases).
+
+### Source
+
+To build `vcsim` from source, first install the [Go toolchain](https://golang.org/dl/).
+
+You can then install the latest `vcsim` from Github using:
+
 ```sh
 % export GOPATH=$HOME/gopath
```

Comment:
There's probably a bunch of places we should drop `GOPATH`, now with go modules.

Reply:
Good point, incl. adding a note on using `vendor`.